app/connectors/enrolments/EnrolmentsConnector.scala

```diff
@@ -24,13 +24,10 @@ import uk.gov.hmrc.http.{HeaderCarrier, SessionKeys}
 import javax.inject.Inject
 import scala.concurrent.{ExecutionContext, Future}
 
-case class EnrolmentsConnector @Inject() (authConnector: AuthConnector)(implicit executionContext: ExecutionContext)
+case class EnrolmentsConnector @Inject() (authConnector: AuthConnector)(using ExecutionContext)
     extends AuthorisedFunctions {
 
-  def maybeAuthenticatedUserEnrolments()(implicit
-    hc: HeaderCarrier,
-    request: Request[_]
-  ): Future[Option[Enrolments]] =
+  def maybeAuthenticatedUserEnrolments()(using request: Request[?])(using HeaderCarrier): Future[Option[Enrolments]] =
```

Comment:
Do previous `implicit` parameter lists now have to be passed through as separate parameter lists?

Reply:
Ah, so we're using the [anonymous givens syntax](https://dotty.epfl.ch/3.0.0/docs/reference/contextual/givens.html#anonymous-givens)? Fine with me, just wondering again if this is a code style choice we want to agree going forward.

Reply:
I like it because it's less boilerplate-y, and it calls out the things are actually being referenced explicitly, plus it saves having to think of (often contrived) names for these context parameters 😅